pkg/hyper/ocicni/ocicni.go

```diff
@@ -29,8 +29,6 @@ import (
 )
 
 type cniNetworkPlugin struct {
```

Comment:
Add a TODO of upgrading cni plugin to its stable release when v0.5.0 is released